app/controllers/submissions_controller.rb

```diff
@@ -308,11 +308,13 @@ def update_files
     end
     unless params[:new_files].nil?
       params[:new_files].each do |f|
-        if f.size > 5000000
+        if f.size > MarkusConfigurator.markus_config_max_file_size
           @file_manager_errors[:size_conflict] =
             "Error occured while uploading file \"" +
              f.original_filename +
-             "\": The size of the uploaded file exceeds the maximum of 5MB."
+             "\": The size of the uploaded file exceeds the maximum of " +
+             "#{(MarkusConfigurator.markus_config_max_file_size/1000000.00).round(2)}" +
+             "MB."
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.